containers/cart-page/index.css

```diff
@@ -0,0 +1,26 @@
+.quantity-section > * {
+  background-color: transparent;
+}
+
+.increase,
+.decrease {
+  width: 2rem;
+  height: 2rem;
+  border: 1.5px solid var(--secondary);
+  text-align: center;
+  font-size: 1.5rem;
+}
+
+.quantity {
+  width: 3rem;
+  height: 2rem;
+  border: 1.5px solid var(--secondary);
```

Comment:
border is having similar properties,  you can make it to separate utility class and use in cart directly